src/components/DeleteTours.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-closing-tag-location */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { deleteTourApi, fetchApiDataTours } from '../redux/tours/toursAPI';
+import '../styles/delete.css';
+
+const DeleteTours = () => {
+  const user = useSelector((store) => store.user);
+  const tours = useSelector((store) => store.tours);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (user.length === 0) {
+      navigate('/login');
+    } else {
+      navigate('/tours/delete');
+    }
+  }, [navigate, user]);
+
+  useEffect(() => {
+    if (tours.length < 1) {
+      navigate('/');
+    }
+  }, [navigate, tours]);
+
+  useEffect(() => {
+    dispatch(fetchApiDataTours());
+  }, [dispatch, fetchApiDataTours, tours]);
+
+  const deleteTour = (id) => {
+    if (user.length > 0) {
+      dispatch(deleteTourApi(id, user[0].token));
+      console.log(tours.length);
```

Comment:
- [ ]  Please, kindly remove this old piece of debugging code from the repo (console.log).  That way, it will be cleaner.  💪